File: app/src/main/java/com/example/simonastojanovic/memeapp/network/ApiInterface.java
package com.example.simonastojanovic.memeapp.network;


import com.example.simonastojanovic.memeapp.model.Response;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface ApiInterface {
    @GET("get_memes")
    Observable<Response> getMemes();

}

File: app/src/main/java/com/example/simonastojanovic/memeapp/app/network/ApiInterface.kt
package com.example.simonastojanovic.memeapp.app.network

import com.example.simonastojanovic.memeapp.app.network.model.Response

import io.reactivex.Observable
import retrofit2.http.GET

interface ApiInterface {

    @get:GET("get_memes")
    val memes: Observable<Response>

}
